datawave-micrometer/src/main/java/datawave/prometheus/DatawavePrometheusMeterRegistry.java

```diff
@@ -62,7 +67,7 @@ public MeterRegistry create(MeterRegistryFactory.InitParameters params) {
             });
 
             new Thread(server::start).start();
-        } catch (IOException e) {
+        } catch (Throwable e) {
             throw new RuntimeException(e);
```

Comment:
It's better to throw a more specific RTE whenever possible. In this case, it's an IOException, so using UncheckedIOException makes sense:

```suggestion
            throw new UncheckedIOException(e);
```